pkg/terraform/exec/plugins/vsphereprivate/resource_vsphereprivate_import_ova.go

```diff
@@ -108,13 +116,15 @@ func findImportOvaParams(client *vim25.Client, datacenter, cluster, datastore, n
 
 	// Find the top-level (and hidden to view) folders in the
 	// datacenter
-	folders, err := importOvaParams.Datacenter.Folders(ctx)
+	//folders, err := importOvaParams.Datacenter.Folders(ctx)
```

Comment:
please don't add commented code. If you want to track something
- it's already in git history
- or should have a jira issue or github issue link to make sure we can track